### ISSUE TITLE
Winetricks Problems

### ISSUE THREAD

**Issue by UltraBlackLinux** [sonic2kk/steamtinkerlaunch#282]
Hey there,
I've been trying to figure this out for quite a long time - STL seems to have some problems with dlls:
If I just select a program to run with Proton, there are no problems at all - Even if it depends on DX10 or DX11.
With STL though, the applications suddenly can't find DX11 anymore (And probably also others), even if Installed via Winetricks.
Now I don't know, what to do anymore - This problem just arose a few days ago, when I was trying to figure out, how to set STL as default compatibility tool.
I already tried to reset the STL game config for that game a few times, with no luck yet.
What am I doing wrong?
Thanks!

**Comment by UltraBlackLinux** [comment]
Did I maybe somehow enable one of [these](https://github.com/frostworx/steamtinkerlaunch/wiki/Compatdata-Redirect)? I can't find a way to toggle them anyways. The settings are very confusing to navigate

**Comment by UltraBlackLinux** [comment]
Oh sorry, I forgot to add the log: [steam-284160.log](https://github.com/frostworx/steamtinkerlaunch/files/6912456/steam-284160.log)

Protontricks: `protontricks (INFO): Found active custom Proton installation: Proton-stl` apparently stl uses a custom proton somehow - This must be breaking it

**Comment by frostworx** [comment]
stl doesn't have problems with dlls, the same problems occur when you start the exact same winetricks commands manually.
Also, this is a duplicate of my "issue" here: https://github.com/frostworx/steamtinkerlaunch/issues/271

Redirecting the compatdata was a feature request [here](https://github.com/frostworx/steamtinkerlaunch/issues/240), but unfortunately OP didn't respond after I implemented it, so I was probably the only one who tested it.
I personally don't use that feature, so PR for fixes and improvements are welcome.
Symlinking compatdata might be related to that problem indeed, but it can also happen when changing proton versions on a compatdata dir.

I don't use protontricks, but "Proton-stl" is the "Internal name of" steamtinkerlaunch itself, as you can see in
`~/.local/share/Steam/compatibilitytools.d/SteamTinkerLaunch/compatibilitytool.vdf` _(or of course in the code directly)_
and no this does not break it.

I haven't debugged more than that what I already wrote into my "[issue](https://github.com/frostworx/steamtinkerlaunch/issues/271)", but it is one of the reasons why I pause development for now as stated [here](https://github.com/frostworx/steamtinkerlaunch/issues/281#issuecomment-889412947)

Any PR which can workaround the problem is very welcome of course.
Closing here, feel free to continue in #271 if you want to help fix that problem

**Comment by UltraBlackLinux** [comment]
> stl doesn't have problems with dlls, the same problems occur when you start the exact same winetricks commands manually.
> Also, this is a duplicate of my "issue" here: #271
> 
> Redirecting the compatdata was a feature request [here](https://github.com/frostworx/steamtinkerlaunch/issues/240), but unfortunately OP didn't respond after I implemented it, so I was probably the only one who tested it.
> I personally don't use that feature, so PR for fixes and improvements are welcome.
> Symlinking compatdata might be related to that problem indeed, but it can also happen when changing proton versions on a compatdata dir.
> 
> I don't use protontricks, but "Proton-stl" is the "Internal name of" steamtinkerlaunch itself, as you can see in
> `~/.local/share/Steam/compatibilitytools.d/SteamTinkerLaunch/compatibilitytool.vdf` _(or of course in the code directly)_
> and no this does not break it.
> 
> I haven't debugged more than that what I already wrote into my "[issue](https://github.com/frostworx/steamtinkerlaunch/issues/271)", but it is one of the reasons why I pause development for now as stated [here](https://github.com/frostworx/steamtinkerlaunch/issues/281#issuecomment-889412947)
> 
> Any PR which can workaround the problem is very welcome of course.
> Closing here, feel free to continue in #271 if you want to help fix that problem

I really don't know, what is going on here at all so I probably won't PR anything - I don't know shell anyways...

Redirecting compatdata is also disabled for me - I couldn't find the setting regardless
